client/src/pages/ProfilePage.jsx

```diff
@@ -1,3 +1,200 @@
+    axios
+      .post(env.CLOUDINARY_API_LINK, formData)
+      .then((response) => {
+        console.log('uploaded picture response: ', response.data);
+        // TODO: make put request to api update profile
+        setDisplayedImg(response.data.url);
+      })
+      .catch((err) => {
+        console.log('error uploading image', err);
+      });
+  };
+  return (
+    <div className="flex justify-center">
+      <DefaultPageLayout>
+        <br />
+        <br />
+        <button
+          className="-ml-5 border border-white"
+          onClick={() => setEditMode(true)}
+        >
+          Edit
+        </button>
+        <button
+          className="ml-25 border border-white"
+          onClick={() => setEditMode(false)}
+        >
+          save
+        </button>
+        <div className="flex ml-10">
+          <div className="relative mr-50">
+            <img
+              src={`${displayedImg}`}
+              alt="Profile-pic"
+              className="w-36 h-36 rounded-full object-cover mr-48"
+            />
+            <img
+              src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg"
+              alt="Flag"
+              className="w-8 h-8 absolute bottom-12 left-28"
+            />
+            {editMode && (
+              <form onSubmit={uploadImage}>
+                <input
+                  type="file"
+                  accept="image/*"
+                  className="block mt-2"
+                  id="profile-pic"
+                  onChange={(event) => {
+                    setSelectedImg(event.target.files[0]);
+                  }}
+                />
+                <button type="submit" className="mt-2 ml-5 border border-white">
+                  upload
+                </button>
+              </form>
+            )}
+          </div>
+          <div className="relative ml-50">
+            <form>
+              {editMode ? (
+                <div className="flex items-center mb-2">
+                  <label htmlFor="name" className="mr-2 font-bold">
+                    Name
+                  </label>
+                  <input
+                    id="name"
+                    type="text"
+                    placeholder="Name"
+                    className="block w-full"
+                  />
+                </div>
+              ) : (
+                <div className="flex items-center mb-2">
+                  <span className="mr-2 font-bold">Name</span>
+                  <span>Emma</span>
+                </div>
+              )}
+
+              {editMode ? (
+                <div className="flex items-center mb-2">
+                  <label htmlFor="@username" className="mr-2 font-bold">
+                    @username
+                  </label>
+                  <input
+                    id="@username"
+                    type="text"
+                    placeholder="@username"
+                    className="block w-full"
+                  />
```

Comment:
Convert to a controlled input like this:
```suggestion
                  <input
                    id="@username"
                    name="username"
                    type="text"
                    placeholder="@username"
                    className="block w-full"
                    value={formData.username}
                    onChange={handleChange}
                  />
```